DownFall/Views/BackpackView.swift

```diff
@@ -174,10 +339,16 @@ class BackpackView: SKSpriteNode {
 
 extension BackpackView {
     override func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
+        touchDelegate?.touchesEnded(touches, with: event)
         guard let touch = touches.first else { return }
         let position = touch.location(in: self)
         for node in self.nodes(at: position) {
-            if let abilityType = node.name {
+            if node.name == targetingAreaName && viewModel.ability != nil && !background.contains(position) {
+               let tileCoord = translatePoint(position)
+               viewModel.didTarget(tileCoord)
+            }
```

Comment:
can we just make the targeting the board, that way we dont have to check if the background contains the touch?

Reply:
I think it would make it a little too complex with maths, leaving as is